native/src/components/TransparentHeader.tsx

```diff
@@ -63,30 +64,29 @@ class TransparentHeader extends React.PureComponent<PropsType> {
         message
       })
     } catch (e) {
-      // TODO Show snackbar
-      reportError(e)
+      const error = e as Error
```

Comment:
Why is this needed?

Reply:
If you do not cast to error explicitley you will get linting erros because  e is of type unknown and reportError() expects an Error as parameter

Reply:
Should only be necessary starting with typescript 4.4

Reply:
So let's leave it for future compatability

Reply:
Actually I'd prefer to remove it again and do this all at once as we probably need this in many places and can find a general solution then.

Reply:
ok, you've got the point